pkg/jwx.go

```diff
@@ -63,7 +63,30 @@ func (client *Crypto) SignJWT(claims map[string]interface{}, key types.KeyIdenti
 		return "", err
 	}
 
-	token, err = SignJWT(rsaPrivateKey, claims)
+	token, err = SignJWT(rsaPrivateKey, claims, nil)
+	return
+}
+
+// SignJWTRFC003 sign a JWT according to Nuts RFC003. This func is only for signing the bearer token of the oauth flow.
+func (client *Crypto) SignJWTRFC003(claims map[string]interface{}, key types.KeyIdentifier) (token string, err error) {
```

Comment:
we're providing a key parameter but not using it? bit confusing. If you really don't want to use it you can call it udnerscore (`_`)

Reply:
oversight